src/codeu/chat/server/Server.java

```diff
@@ -192,13 +197,19 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
     this.commands.put(NetworkCode.CLEAN_REQUEST, new Command() {
       @Override
       public void onMessage(InputStream in, OutputStream out) throws IOException {
-        File log = new File(persistentPath.getPath());
+        File log = new File(persistentPath, "log.txt");
         FileWriter writer = new FileWriter(log);
         writer.write("");
         writer.close();
 
+        File passwords = new File(persistentPath, "passwords.txt");
```

Comment:
have the controller do this. the same old idea: all logic should be concentrated in as few classes as possible